### DISCUSSION TITLE
Internal Baro died, trying to add external.

### DISCUSSION THREAD

**Discussion by kaliblazin707** [iNavFlight/inav#7741] [Q&A] [answered]
My internal Barometer died on my MatekF411WSE. (SLP revision)
It still shows blue in the configurator, however sensors show 0.0. 
I bought some BMP280s on Amazon. However, when I hook it up and select BMP280 in the config tab, its red. reads 0.0
Auto defaults to the dead internal one. 
Is there any way to like, CLI tell the internal one to piss off?
Could the external one just be dead?
VCC to 3.3
GND to GND
SDA to SDA
SCL to SCL

**Comment by kaliblazin707** [comment]
Any suggestions would be appreciated

**Comment by avsaase** [comment, marked as answer]
You need to physically remove the dead baro or cut the trace on the pcb so it doesn't get power.

**Reply by kaliblazin707** [reply]
Ty Sir!
